src/app/domain/case-viewer/components/case-viewer/case-viewer.component.html

```diff
@@ -4,8 +4,8 @@
             <ng-container *ngFor="let section of targetSection.sections">
                 <ng-container [ngSwitch]="section.type" >
                     <h1 data-selector="heading" class="govuk-heading-l">{{section.name}}</h1>
-                    <app-summary-panel data-selector="summary-panel" *ngSwitchCase="'summary-panel'" [panelData]="section"></app-summary-panel>
-                    <app-document-panel data-selector="document-panel" *ngSwitchCase="'document-panel'" [panelData]="section"></app-document-panel>
+                    <app-summary-panel data-selector="summary-panel" *ngSwitchCase="'summary-panel'" [panelData]="section" [caseId]="case.caseId"></app-summary-panel>
+                    <app-document-panel data-selector="document-panel" *ngSwitchCase="'document-panel'" [panelData]="targetSection.sections"></app-document-panel>
```

Comment:
Any reason this got changed? DOesn't it break the document panel?

Reply:
fixed